bin/wp.bat

```diff
@@ -0,0 +1,20 @@
+@ECHO OFF
+IF "%APP_PATH%"=="" (
+  ECHO Error: APP_PATH is undefined; Studio's bundled 'wp-cli' is only usable in shell sessions originating from Studio's "Open in Terminal" feature.
+  EXIT /B 1
+)
+
+SET COMMAND=%*
+SET CLI=wp %COMMAND%
```

Comment:
I noticed that running commands that include quotes in the arguments don't work on Windows. I tried different approaches but nonce seemed to work.